.github/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,13 @@
 <!--
 Please fill out this template with all the information you have. We can't do much without
-both logs and a detailed description of what you encountered. Please do your best!
+both the logs and a detailed description of what you've encountered. Please do your best!
 
 Please note that this tracker is only for bugs and feature requests. Please try these
 locations if you have a question or comment:
 
   https://community.signalusers.org/
   http://support.signal.org/
-  support@signal.org
+  mailto:support@signal.org
```

Comment:
Please remove this.

Reply:
Done! removed it, only thought it would be more reachable by one click.